.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: node_js
 node_js:
-  - 4
-  - 6
-  - 7
+  - 8
```

Comment:
8 is EOL, feel free to 🔥

Reply:
Travis told me that by failing the build :D